tests/testAutopas/tests/iterators/ParticleIteratorInterfaceTest.cpp

```diff
@@ -192,22 +162,24 @@ TEST_P(ParticleIteratorInterfaceTest, findAllParticlesAroundBoundaries) {
   // set up expectations
   std::vector<size_t> expectedIDs;
   switch (behavior) {
-    case autopas::IteratorBehavior::ownedOnly: {
+    case autopas::IteratorBehavior::owned: {
       expectedIDs = particleIDsOwned;
       break;
     }
-    case autopas::IteratorBehavior::haloOnly: {
+    case autopas::IteratorBehavior::halo: {
       expectedIDs = particleIDsHalo;
       break;
     }
-    case autopas::IteratorBehavior::haloAndOwned: {
+    case autopas::IteratorBehavior::ownedOrHalo: {
       expectedIDs = particleIDsOwned;
       expectedIDs.insert(expectedIDs.end(), particleIDsHalo.begin(), particleIDsHalo.end());
       break;
     }
-    case autopas::IteratorBehavior::haloOwnedAndDummy: {
-      GTEST_FAIL() << "IteratorBehavior::haloOwnedAndDummy should not be tested through this test"
-                      " as container behavior with dummy particles is not uniform.";
+    default: {
+      GTEST_FAIL() << "IteratorBehavior::" << behavior
+                   << "  should not be tested through this test!\n"
+                      "Container behavior with dummy particles is not uniform.\n"
+                      "forceSequential alone makes no sense.";
```

Comment:
```suggestion
                      "Using forceSequential is not supported.";
```
even if `forceSequential` is not specified alone, the default branch will be hit.